clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -3061,22 +3080,7 @@ static void RenderFloatingPointOptions(const ToolChain &TC, const Driver &D,
         continue;
       [[fallthrough]];
```

Comment:
Should fallthrough still be used?

Reply:
This is falling through from OPT_Ofast to OPT_ffast_math. I think we still want that to happen. It's not obvious from the diff, but the "fp-model" handler and the "ffast-math" handler are in different switch statements.

Reply:
Oh! Thank you for mentioning they're in different `switch` statements, I had missed that as well. :-D